packages/dates/src/utilities/tests/formatDate.test.ts

```diff
@@ -0,0 +1,22 @@
+import {formatDate} from '../formatDate';
+
+jest.mock('../../map-deprecated-timezones', () => ({
+  mapDeprecatedTimezones: jest.fn(),
+}));
+
+const mapDeprecatedTimezones: jest.Mock = jest.requireMock(
+  '../../map-deprecated-timezones',
+).mapDeprecatedTimezones;
+
+describe('formatDate', () => {
+  it('maps the deprecated timezone', () => {
+    const date = new Date();
+    const locale = 'en-UA';
+    const options: Intl.DateTimeFormatOptions = {
+      timeZone: 'Europe/Kyiv',
+    };
+
+    formatDate(date, locale, options);
+    expect(mapDeprecatedTimezones).toHaveBeenCalledWith('Europe/Kyiv');
```

Comment:
THIS TEST ISN'T GREAT ... but like at least we're testing that the function is actually being called, AND the fact that `quilt/packages/dates/src/tests/map-deprecated-timezones.test.ts` is testing that it's mapping / converting, feels safe to me - Ideally mocking out a real "bug" would be great here, but since we can't ... this is good enough imo 😄 ❤️